contract_invoice_merge_by_partner/models/account_analytic_analysis.py

```diff
@@ -0,0 +1,31 @@
+# -*- coding: utf-8 -*-
+# © 2016 Carlos Dauden <carlos.dauden@tecnativa.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from openerp import api, models
+
+
+class PurchaseOrderLine(models.Model):
+    _inherit = 'account.analytic.account'
+
+    @api.multi
+    def _recurring_create_invoice(self, automatic=False):
+        invoice_obj = self.env['account.invoice']
+        invoices = invoice_obj.browse(
```

Comment:
I don't return invoices var.
If you are referring to super, it seems that always return a list or IDs https://github.com/odoo/odoo/blob/8.0/addons/account_analytic_analysis/account_analytic_analysis.py#L791